api/v1alpha1/openapi.yaml

```diff
@@ -3031,6 +3031,56 @@ components:
               description: 'The directory in which the executable will be run from if it is left empty it will run from the users home directory.'
           required:
             - run
+    DeviceUpdatePolicySpec:
+      type: object
+      properties:
+        download:
+          $ref: '#/components/schemas/UpdatePolicy'
+        update:
+          $ref: '#/components/schemas/UpdatePolicy' 
+          description: "Defines the policy for downloads and updates, including schedules and rate limits."
+    UpdatePolicy:
+      type: object
+      properties:
+        schedule:
+          $ref: '#/components/schemas/UpdateSchedule'
+      description: "A policy defining the schedule for downloads or updates."
+    UpdateSchedule:
+      type: object
+      properties:
+        timezone:
+          type: string
+          description: "Timezone for the schedule (e.g., GMT) the defaults to the local system timezone"
```

Comment:
Do we expect time zones to be specified by the tzdata identifier (e.g. America/New_York) or abbreviation (e.g. EST, EDT)? The latter are ambiguous.

Should we default the local timezone instead of making it optional (avoid pointer)? Should default be an explicit “local” instead of “”?

Reply:
will align with https://data.iana.org/time-zones/theory.html#naming